flang/test/Semantics/OpenMP/omp-common-fp-lp.f90

```diff
@@ -0,0 +1,13 @@
+! RUN: %flang_fc1 -fopenmp -fsyntax-only %s 2>&1 | FileCheck %s --allow-empty
```

Comment:
Please could you add -fopenmp-version=60 (or whichever version was the first to allow this) so that this test does not fail once the required check is added for older openmp versions.